src/main/kotlin/team/sfe/server/domain/user/presentation/UserController.kt

```diff
@@ -0,0 +1,42 @@
+package team.sfe.server.domain.user.presentation
+
+import jakarta.validation.Valid
+import org.springframework.http.HttpStatus
+import org.springframework.web.bind.annotation.PatchMapping
+import org.springframework.web.bind.annotation.PostMapping
+import org.springframework.web.bind.annotation.RequestBody
+import org.springframework.web.bind.annotation.RequestHeader
+import org.springframework.web.bind.annotation.RequestMapping
+import org.springframework.web.bind.annotation.ResponseStatus
+import org.springframework.web.bind.annotation.RestController
+import team.sfe.server.domain.user.presentation.request.UserSignInRequest
+import team.sfe.server.domain.user.presentation.request.UserSignUpRequest
+import team.sfe.server.domain.user.presentation.response.TokenResponse
+import team.sfe.server.domain.user.service.TokenRefreshService
+import team.sfe.server.domain.user.service.UserSignInService
+import team.sfe.server.domain.user.service.UserSignUpService
+
+@RequestMapping("/user")
+@RestController
+class UserController(
+    private val userSignUpService: UserSignUpService,
+    private val userSignInService: UserSignInService,
+    private val tokenRefreshService: TokenRefreshService
+) {
+
+    @ResponseStatus(HttpStatus.CREATED)
+    @PostMapping("/signup")
+    fun signUp(@RequestBody request: @Valid UserSignUpRequest) {
```

Comment:
```suggestion
    fun signUp(@RequestBody @Valid request: UserSignUpRequest) {
```
위에가 되는지 모르겠는데 보통 이렇게 사용합니다!